### ISSUE TITLE
Mantisshrimp Milestones

### ISSUE THREAD

**Issue by ai-fast-track** [airctic/mantisshrimp#135]
# 🚀 Milestones

This lists the pre-requisites for our **Milestones**:

## Milestone 1
**Parsers**: Implementing the following parsers
	- [x] COCO Parser
	- [x] VOC Parser

**DL Engines**: Bridging Mantisshrimp to different DL libraries
	- [x] Fastai Integration
	- [x] Pytorch-Lightning Integration

**Models**: 
	- [x] FasterRCNN
	- [x] MantisMaskRCNN

**Backbones**: Connecting different backbones to Mantisshrimp
	- [x] Backbone Generic 
	- [x] Backbone Adaptive 
	- [x] Backbone Features 
	- [x] torchvision Connection

**Models Hub**: Loose coupling with different models implementation through referencing to original models repos
	- [x] Detr

**Parsers Hub**: Community Parsers
	- [x] Wheat Parser
	- [x] PennFundanParser

**Datasets**: List of datasets used in tutorials
	- [x] Pets
	- [x] Wheat
	- [ ] Fridge objects

**Tutorials**: 
	- [ ] Getting started: FasterRCNN + Pets dataset + Fastai + Pytorch-Lightning + Visualization (showing images, boxes, and labels
	- [ ] Advanced Tutorial: similar to the previous but using Wheat dataset + uses different backbones
	- [ ] How to train your own dataset: Step by step tutorial - Uses Fridge objects dataset

**Documentation**: 
	- [ ] Documentation generated using  `.py` modules
	- [ ] Documentation generated using the different tutorials notebooks
	- [ ] Creating some visual illustrations

## Milestone 2
It is Milestone 1 + the following pre-requisites:

**Deployment**: 
	- [ ] Using Streamlit: Locally + in the cloud

**Datsets**: Add other datasets such the following:
	- [ ] Blood cells dataset (BCCD)
	- [ ] Plant disease
	
**Logging**: Integrating some logging tools
[ ] Wandb

**Comment by lgvaz** [comment]
Can you create this as a project? (or as a milestone even, I just find projects more organized because of the cards)

**Comment by lgvaz** [comment]
Moved to the #Milestone1 project
